pycds/orm/tables.py

```diff
@@ -8,6 +8,12 @@
 ORM class via `ORMClass.__table__.indexes`. The latter is very convenient, so
 we make sure always to declare indexes outside of classes. See code below for
 many examples.
+
+2. Relationships should be declared using `back_populates=`, and *not* using
```

Comment:
This is maybe a bit picky, but might we say that "we are choosing to use `back_populates=`..." rather than "should"?

Reply:
Yes, it's a choice. Good point.